### PR TITLE
Updated the gamerules in game-board 

### PR DESCRIPTION
**Pull request by Dewonglucas11**
I have updated the rules i added previously to be able to handle requests from the foundation as well as the tableau stacks.

### CODE REVIEW

File: src/game-board/game-board.html [hunk 1]
```diff
@@ -60,15 +60,24 @@
         //1 to avoid an out of bounds exception.
         var stackCardRank = rankList.indexOf(stack[stack.length-1].rank);
         //Simple comparison here. If the the card we want to place is a rank lower than the card on top of the stack (suit doesn't matter here)
-        //Then we return true. Else we return false. 
-        if(cardRank == stackCardRank - 1){
-          return true;
-        } else{
-          return false;
+        //Then we return true. Else we return false.
+        if(isFoundation == true){
```
Comment (zacharytamas) [inline src/game-board/game-board.html]: The strange indentation throughout this bit makes it harder to understand.

**Comment by zacharytamas** [comment]
Would have preferred that you'd not committed some of this directly to master without letting rest of team review it.

I think I would have put the logic for whether a stack accepts a drop to it on the `card-stack` itself but if this is already actually working it's likely fine here given our deadline.

**Comment by zacharytamas** [comment]
👍

File: src/game-board/game-board.html [hunk 2]
```diff
@@ -60,15 +60,24 @@
...
+          if(cardRank == stackCardRank - 1){
+            return true;
```
Comment (anthonylee83) [inline src/game-board/game-board.html]: Unless I missed it elsewhere we also need to check to make sure that the suit is the same for foundation stacks.

File: src/game-board/game-board.html (hunk 1 above)
Comment (Dewonglucas11) [reply src/game-board/game-board.html]: Mb, I thought I had correct indentation in Atom, but I think it translated weirdly over here. In any case I think it is ready to merge now that I added additional logic to account for identical suits in the foundation stacks

**Pull request merged**